src/Sarif/Core/Tool.cs

```diff
@@ -15,15 +15,15 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// </summary>
     public partial class Tool 
     {
-        public static Tool CreateFromAssemblyData(string prereleaseInfo = null)
+        public static Tool CreateFromAssemblyData(string prereleaseInfo = null, string friendlyName = null)
```

Comment:
SARIF is not a CLSCompliant assembly. See AssemblyInfo.cs. I believe that the prominent managed languages of interest for the SDK now support default arguments (the original prohibition on default arguments, in fact, was based on the lack of C# support, as named parameters has always been a VB thing).